pynetdicom/tests/test_acse.py

```diff
@@ -2771,18 +2775,29 @@ def handle(event):
 
         self.ae = ae = AE()
         ae.add_requested_context(VerificationSOPClass)
+        ae.add_supported_context(VerificationSOPClass)
```

Comment:
This test was relying on the `acse_timeout` to generate an abort. I rewrote the test to simulate an explicit abort from the scp.